app/src/main/java/cm/aptoide/pt/home/BottomNavigationView.kt

```diff
@@ -0,0 +1,103 @@
+package cm.aptoide.pt.home
+
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import cm.aptoide.pt.home.appcoins.BonusScreen
+import cm.aptoide.pt.home.apps.AppsScreen
+import cm.aptoide.pt.home.games.GamesScreen
+import cm.aptoide.pt.home.search.SearchScreen
+import cm.aptoide.pt.home.updates.UpdatesScreen
+import cm.aptoide.pt.theme.AppTheme
+
+@Preview
+@Composable
+fun MainView() {
+  val navController = rememberNavController()
+  Scaffold(
+      bottomBar = {
+        BottomNavigation(navController)
+      }
+  ) {
+    NavigationGraph(navController)
+  }
+}
+
+@Composable
+private fun BottomNavigation(navController: NavHostController) {
+  val items = listOf(
+      BottomNavigationMenus.Games,
+      BottomNavigationMenus.Apps,
+      BottomNavigationMenus.AppCoins,
+      BottomNavigationMenus.Search,
+      BottomNavigationMenus.Updates
+  )
+  CompositionLocalProvider(LocalElevationOverlay provides null) {
+    BottomNavigation(backgroundColor = AppTheme.colors.surface) {
+      val navBackStackEntry by navController.currentBackStackEntryAsState()
+      val currentDestination = navBackStackEntry?.destination
+      items.forEach { screen ->
+        val selected = currentDestination?.hierarchy?.any { it.route == screen.route }
+        BottomNavigationItem(
+            icon = { Icon(imageVector = screen.icon, contentDescription = null) },
+            selected = selected == true,
```

Comment:
why not just selected ?

Reply:
selected is nullable, this is the most elegant way of making sure it does not crash when null (== run equals which checks for null) without adding so much boilerplate code or using non-safe operations like !!